features/interest-invokers.yml

```
@@ -0,0 +1,4 @@
+name: 'Interest invokers'
+description: The `interestfor` global HTML attribute triggers actions on a target element when the user shows interest (e.g., hover, focus, long-press). The user agent detects this interest, firing an InterestEvent on the target. This event has default actions, like showing or hiding popovers.
```

(Assuming the description, as written, is accurate.) Optional style nits/suggestions:

```suggestion
description: The `interestfor` global HTML attribute triggers actions on a target element when the user shows interest (such as hover, focus, or long-press). If the browser detects this interest, it fires an `InterestEvent` on the target. This event has default actions, like showing or hiding popovers.
```